src/measure_theory/measurable_space.lean

```diff
@@ -80,19 +80,19 @@ variable [measurable_space α]
 /-- `is_measurable s` means that `s` is measurable (in the ambient measure space on `α`) -/
 def is_measurable : set α → Prop := ‹measurable_space α›.is_measurable
 
-lemma is_measurable.empty : is_measurable (∅ : set α) :=
+@[simp] lemma is_measurable.empty : is_measurable (∅ : set α) :=
 ‹measurable_space α›.is_measurable_empty
 
-lemma is_measurable.compl : is_measurable s → is_measurable sᶜ :=
+@[simp] lemma is_measurable.compl : is_measurable s → is_measurable sᶜ :=
```

Comment:
Remove the simp attribute on this one, as `is_measurable.compl_iff` should be enough?